freshmaker/config.py

```diff
@@ -401,6 +399,16 @@ class Config(object):
             'type': str,
             'default': 'all',
             'desc': 'vcr mode for recording lightblue queries'
+        },
+        'pyxis_server_url': {
```

Comment:
Could you add a validation function in this module (`def _pyxis_server_url(self, s):`) so that you raise a `ValueError` if the `HandleBotasAdvisory` handler is enabled but this configuration is not defined?

This will halt Freshmaker from starting at all if it's improperly configured.

Reply:
Hmm but what if we will use that module(Pyxis) in other handlers?

Reply:
@apaplaus then we also update the `_setifok_pyxis_server_url` function. Alternatively, you could just make this value required all the time.